bundle/manifests/atlas.mongodb.com_atlascustomroles.yaml

```diff
@@ -29,9 +29,6 @@ spec:
     - jsonPath: .spec.role.name
       name: Name
       type: string
-    - jsonPath: .spec.projectIDRef.id
-      name: Project ID
-      type: string
```

Comment:
i was puzzled about this but this is correct, external project ref is optional so it should not be displayed as a printer column. @igor-karpukhin fixed this.

Reply:
This is because the bundle wasn't updated with the fix. But this is okay, we're have bundle (and CRDs) regenerated every time when we release